### PR TITLE
User avatar is now shown for people without PRs

### PR DESCRIPTION
**Pull request by Metamist**
# Fixes :shipit: :shipit: :shipit:
- https://github.com/jenkoian/hacktoberfest-checker/issues/140

# Summary :1st_place_medal: :1st_place_medal: :1st_place_medal:
This pull request will allow the web interface to show data about the user, which currently is only used for the avatar.
Previous method was by getting the avatar from the pull requests, but if none are found the user avatar would not be shown.
The `findPrs` function is now a bit more generalized by returning the prs and the user data as resolved promise parameters, rather than variables defined inside the request.

Some suggestions would be to generalize the `findPrs` even further by moving it out of the request (thus easing the load on the app each request) and possibly exporting it as well.

### CODE REVIEW

File: public/js/main.js [hunk 1]
```diff
@@ -101,7 +101,7 @@ HacktoberfestChecker.prototype.makeSpinner = function() {
  */
 HacktoberfestChecker.prototype.makeError = function(error) {
     return $('<div/>').append(
-        $('<h2/>', {
+        $('<h2 class="orange"/>', {
```
Comment (jenkoian) [inline public/js/main.js]: This has since been changed to be white, so can probably lose this?

**Comment by jenkoian** [comment]
Thakns @Metamist ! Hopefully it won't affect rate limits (as will now be making twice the calls) so will keep an eye on it, but a great feature 👍

**Comment by Metamist** [comment]
The github client is never authenticated with the github token provided in the `.env` file. It is using the public api, thus no rate limits! :+1:

File: public/js/main.js (hunk 1 above)
Comment (Metamist) [reply public/js/main.js]: Did not mean to include this in this PR. Seems to have worked out anyway, my bad nevertheless. :)

**Comment by jenkoian** [comment]
You are right! That seems like a bug though, the search api does appear to have rate limits? https://developer.github.com/v3/search/#rate-limit

**Comment by jenkoian** [comment]
@Metamist fancy taking a quick glance at https://github.com/jenkoian/hacktoberfest-checker/pull/154 for me?

**Comment by Metamist** [comment]
Actually seems like I was wrong. Fetching a user does also count towards the rate limit. Seems strange to me, as I read somewhere that using api that display data that is publicly visible on the website shouldn't count towards the rate limit.

**Pull request merged**